documentation/src/developer_documentation/chain_muxer_state_machine.md

```diff
@@ -0,0 +1,42 @@
+`ChainMuxer` state transitions:
+
+```mermaid
+flowchart TD
+    A[Idle]
+    B[Connect]
+    C[Bootstrap]
+    D[Follow]
+
+    A -->|sync| B
+    A -->|skip| D
```

Comment:
Could you please describe what makes `ChainMuxer` go this path?

Reply:
It can be manually told to skip directly to `follow`. We never do it and I don't understand why it is there.